loopin.go

```diff
@@ -189,6 +190,24 @@ func newLoopInSwap(globalCtx context.Context, cfg *swapConfig,
 		return nil, err
 	}
 
+	// Default the HTLC internal key to our sender key.
+	senderInternalPubKey := senderKey
+	var senderInternalPrivKey [32]byte
+
+	// If this is a MuSig2 swap then we'll generate a brand new key pair
+	// and will use that as the internal key for the HTLC.
+	if loopdb.CurrentProtocolVersion() >= loopdb.ProtocolVersionMuSig2 {
```

Comment:
What does the `>=` imply for the order of protocol versions? Anything that comes after `ProtocolVersionMuSig2 ` is gonna share the same code in the block below? If yes, why?

Reply:
This just means that currently we make the assumption that all further protocol upgrades will include musig2 loopin. If this is not the case we'll simply change it.